engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/testing/ProcessManagerStub.scala

```diff
@@ -0,0 +1,52 @@
+package pl.touk.nussknacker.engine.testing
+
+import com.typesafe.config.Config
+import pl.touk.nussknacker.engine.api.deployment.simple.SimpleProcessStateDefinitionManager
+import pl.touk.nussknacker.engine.api.{ProcessVersion, StreamMetaData, TypeSpecificData}
+import pl.touk.nussknacker.engine.api.deployment.{CustomAction, CustomActionError, CustomActionNotImplemented, CustomActionRequest, CustomActionResult, ProcessDeploymentData, ProcessManager, ProcessState, ProcessStateDefinitionManager, SavepointResult, TestProcess, User}
+import pl.touk.nussknacker.engine.api.process.ProcessName
+import pl.touk.nussknacker.engine.{ModelData, ProcessManagerProvider}
+import pl.touk.nussknacker.engine.api.queryablestate.QueryableClient
+
+import scala.concurrent.Future
+
+class ProcessManagerStub extends ProcessManager {
```

Comment:
I wonder if we could use it from docker e.g. for purpose of cypress tests or in case if someone don't want to setup flink and don't want to see nasty errors during fetch status

Reply:
I think we can, I added it also to ServiceLoader - loading is by name so it should be ok